app/main.py

```diff
@@ -1 +1,7 @@
 # write your code here
+def copy_file(command):
+    if not command.split()[1] == command.split()[2]:
```

Comment:
check what command is provided

Reply:
comment is not clear, check where? tests file is empty in this task